### PR TITLE
bigquery: add a bigquery_connection_quickstart

### PR DESCRIPTION
**Pull request by shollyman**
First cut at a bigquery connection quickstart.  A more exhaustive
example necessitates further coordination regarding the cloud sql
control plane API.

### CODE REVIEW

**Comment by shollyman** [comment]
Adding Tyler as well.  Tyler, this is another new quickstart and we need to do the API enablement for bigqueryconnection.googleapis.com on projects in the test pool.

File: bigquery/bigquery_connection_quickstart/main.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	connectionpb "google.golang.org/genproto/googleapis/cloud/bigquery/connection/v1"
+)
+
+func main() {
+
+	// Define two command line flags for controlling the behavior of this quickstart.
+	var (
+		projectID = flag.String("project_id", "", "Cloud Project ID, used for session creation.")
+		location  = flag.String("location", "US", "BigQuery location used for interactions.")
+	)
```
Comment (tbpg) [inline bigquery/bigquery_connection_quickstart/main.go:40]: Optional:
```suggestion
func main() {
	// Define two command line flags for controlling the behavior of this quickstart.
	projectID := flag.String("project_id", "", "Cloud Project ID, used for session creation.")
	location  := flag.String("location", "US", "BigQuery location used for interactions.")
```

File: bigquery/bigquery_connection_quickstart/main.go [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+			return "", err
+		}
+		fmt.Fprintf(&buf, "\tConnection %s was created %s\n", conn.GetName(), unixMillisToTime(conn.GetCreationTime()).Format(time.RFC822Z))
+		totalConnections++
+	}
+	fmt.Fprintf(&buf, "\n%d connections processed.\n", totalConnections)
+	return buf.String(), nil
+}
+
+// A simple helper function to convert the epoch-millisecond representations used by the API into a time.Time representation.
```
Comment (tbpg) [inline bigquery/bigquery_connection_quickstart/main.go:89]:
```suggestion
// unixMillisToTime converts the epoch-millisecond representations used by the API into a time.Time representation.
```

File: bigquery/bigquery_connection_quickstart/main.go (hunk 1 above)
Comment (shollyman) [reply bigquery/bigquery_connection_quickstart/main.go]: done

**Pull request merged**